### PR TITLE
Updates single quotes to apostrophes where appropriate

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

 If merged, this PR changes some instances of single quotes to apostrophes or double quotes where appropriate. Also fixes some minor spelling errors.

### CODE REVIEW

File: website/docs/components/form/toggle/partials/code/how-to-use.md [hunk 1]
```diff
@@ -146,11 +146,11 @@ You can use different events, depending on your context/need (eg. `input`, `chan
...
 
-It's unlikely that you will ever need to use a "group" of toggle fields (from a design/UX perspective in this case is better to use a group of checkboxes). For this reason we will not explain in detail how to use the "group" variant (apart from the case of a toggle "group" with single choice). Please refer to the documentation for the [`Form::Toggle::Group`](/components/form/checkbox/) documentation for more details, or speak with the design system team for guidance on what to do in this specific case.
+it’s unlikely that you will ever need to use a "group" of toggle fields (from a design/UX perspective in this case is better to use a group of checkboxes). For this reason we will not explain in detail how to use the "group" variant (apart from the case of a toggle "group" with single choice). Please refer to the documentation for the [`Form::Toggle::Group`](/components/form/checkbox/) documentation for more details, or speak with the design system team for guidance on what to do in this specific case.
```
Comment (jorytindall) [inline website/docs/components/form/toggle/partials/code/how-to-use.md:149]:
```suggestion
It’s unlikely that you will ever need to use a "group" of toggle fields (from a design/UX perspective in this case is better to use a group of checkboxes). For this reason we will not explain in detail how to use the "group" variant (apart from the case of a toggle "group" with single choice). Please refer to the documentation for the [`Form::Toggle::Group`](/components/form/checkbox/) documentation for more details, or speak with the design system team for guidance on what to do in this specific case.
```

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ In most cases use a medium size, but use a link size that best fits the UI.
 
 For example:
 
-- Don't use large links in tables.
+- don’t use large links in tables.
```
Comment (jorytindall) [inline website/docs/components/link/standalone/partials/guidelines/guidelines.md:25]: Probably best to address this in the future, but there are a bunch of instances of single bulleted lists which are a bit counter-intuitive.

File: website/docs/components/modal/partials/guidelines/guidelines.md [hunk 3]
```diff
@@ -221,7 +221,7 @@ This action may impact areas of the application outside of the scope of the curr
 <Hds::Modal::Footer>
   <Hds::ButtonSet>
     <Hds::Button @text="Rename this cluster" />
-    <Hds::Button @text="Don't rename this cluster" @color="secondary" />
+    <Hds::Button @text="don’t rename this cluster" @color="secondary" />
```
Comment (jorytindall) [inline website/docs/components/modal/partials/guidelines/guidelines.md:224]:
```suggestion
    <Hds::Button @text="Don’t rename this cluster" @color="secondary" />
```

File: website/docs/components/modal/partials/guidelines/guidelines.md [hunk 4]
```diff
@@ -112,15 +112,15 @@ Large
   </M.Footer>
 </Hds::Modal>
 
-**warning** indicates that the Modal's action may impact a setting, item, or object and is used to message the potential impact while confirming that the user wants to continue.
+**warning** indicates that the Modal’s action may impact a setting, item, or object and is used to message the potential impact while confirming that the user wants to continue.
```
Comment (jorytindall) [inline website/docs/components/modal/partials/guidelines/guidelines.md:115]:
```suggestion
**Warning** indicates that the Modal’s action may impact a setting, item, or object and is used to message the potential impact while confirming that the user wants to continue.
```

**Review by jorytindall — APPROVED** [review]
Couple of instances where it looks like the capitalization was changed (maybe overwritten?) Feel free to disregard, I wasn't looking at everything contextually.

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md (hunk 2 above)
Comment (MelSumner) [reply website/docs/components/link/standalone/partials/guidelines/guidelines.md:25]: Let's chat in Slack about this, I was also curious about how we should handle this.

**Pull request merged**